src/collective/cover/browser/scaling.py

```diff
@@ -158,10 +158,12 @@ def modified(self):
                                             name='images',
                                             default=None)
             return base_scales and base_scales.modified()
-        mtime = ''
+        mtime = None
         for k, v in self.context.data.items():
             if INamedImage.providedBy(v):
-                mtime += self.context.data.get('{0}_mtime'.format(k), '')
+                image_time = self.context.data.get('{0}_mtime'.format(k), None)
+                if mtime is None or image_time > mtime:
```

Comment:
The code at tiles/data.py is made to search any image field and set the modified date into a data attribute.
I think this is done this way because there is no way to know if someone creates a tile with more than one image.

On the other hand, the timestamp setted at the same code will be the same for all image items.. so we could cut the complexity and create just one attribute for every image, removing the need to make a loop and check if the item is INamedImage.

I don't know if a refactory at this code is beyond the borders of this pull request, or if this is a good time to do it.